CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-20223: Added a new default policy category called "Zero Trust", and the following default policies have been tagged with the category: "Deployments should have at least one ingress Network Policy" and "Unauthorized Network Flow"
 - ROX-20224: Added a new default policy category called "Supply Chain Security", and the following default policies have been tagged with the category: "Images with no scans", "30-day Scan Age", "90-day Image Age", "Required Annotation: Email",
   "Required Annotation: Owner/Team", "Required Label: Owner/Team" and "Latest tag"
+- ROX-21207: All usage of RocksDB has been removed.
+  - As mentioned in previous deprecation notice, users running ACS version 3.74.x or earlier must stop at version 4.4.x before upgrading to 4.5 or later. In version 4.0.0, ACS switched the underlying datastore to PostgreSQL. On an upgrade, data would be automatically migrated to PostgresSQL from the previous datastore.
```

Comment:
How about

We deprecate the support to directly upgrade from any ACS version 3.74.x or earlier to version 4.5. ACS version 3.74.x or earlier should be upgraded to version 4.4.x first.